Sources/swift-test/test.swift

```diff
@@ -31,7 +32,7 @@ func test(path: String, xctestArg: String? = nil) throws -> Bool {
     }
 #endif
 
-    let result: Void? = try? system(args)
+    let result: Void? = try? system(args, environment: NSProcessInfo.processInfo().environment)
```

Comment:
That would be the same as passing no environment (nil) to either posix_spawn or NSTask.environment, wouldn't that be cleaner?